src/index.html

```diff
@@ -3,8 +3,20 @@
 <head>
     <meta charset="UTF-8">
     <title>Country Information - World Regions</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">
```

Comment:
Waarom heb je hier geopteerd voor een pre-connect vs. het enkel 'simpelweg' importeren van een font in het css bestand?